test/check-docker

```diff
@@ -54,10 +54,18 @@ class TestDocker(MachineCase):
         # https://github.com/docker/docker/issues/10617
         b.wait_in_text("#container-terminal", message)
 
+    def fix_docker_config(self):
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1281552
+        # The 'journald' logdrive is used by default although it
+        # doesn't allow reading via 'docker logs'.  So we kick it out of the config.
+        self.machine.execute("sed -i -e 's/--log-driver=journald//' /etc/sysconfig/docker")
```

Comment:
I think we should use a known issue instead. Users will hit this exact problem when they try to use Cockpit.